cookie-banner-rules-list.json

```diff
@@ -7476,6 +7476,1182 @@
         ]
       },
       "id": "1bf6fa4e-7882-41cf-b5e1-ef3c91485006"
+    },
+    {
+      "click": {
+        "optIn": "button#cookie-banner-initial-accept-all",
+        "presence": "div.cookie-initial-buttons"
+      },
+      "domain": "swedbank.se",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "SSLB",
+            "value": "1"
+          },
+          {
+            "name": "s_cc",
+            "value": "true"
+          }
+        ],
+        "optOut": []
+      },
+      "id": "f3afe889-1fbe-4c24-b62d-ca1bd6c5f636"
+    },
+    {
+      "click": {
+        "optIn": "button#consent_prompt_submit",
+        "presence": "div.consent_prompt_footer"
+      },
+      "domain": "argos.co.uk",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_tt_enable_cookie",
+            "value": "1"
+          }
+        ]
+      },
+      "id": "88e79126-8072-4079-aa60-1d71d9ddb65b"
+    },
+    {
+      "click": {
+        "optIn": "button#onetrust-accept-btn-handler",
+        "presence": "div.banner-actions-container"
+      },
+      "domain": "gutekueche.at",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_pbjs_userid_consent_data",
+            "value": "618401108686130"
```

Comment:
It looks to me that this value is related to user ID from its name. Maybe we shouldn't include a cookie that could be derived from a user id in our list. @Trikolon wdyt?

Reply:
Agreed, I'd prefer using a click rule for  this case. It's a sourcepoint frame so this should work:
```
[{
  "domain": "gutekueche.at",
  "click": {
    "optIn": ".sp_choice_type_11",
    "presence": ".message-container >  #notice",
    "skipPresenceVisibilityCheck": true
  },
  "id": "58dd5020-ecc0-4fdc-813b-1009fa83794a"
}]
```

Reply:
Replaced the rule with the one provided by @Trikolon.